TECHMANIA/Assets/Scripts/Components/Editor Scene/TrackSetupPanel.cs

```diff
@@ -138,19 +138,19 @@ public void OnImportButtonClick()
                 if (i == 10)
                 {
                     fileList += "\n";
-                    fileList += Locale.GetStringAndFormat(
+                    fileList += Locale.GetStringAndFormatIncludingPaths(
                         "track_setup_resource_tab_overwrite_omitted_files",
                         filesToBeOverwritten.Count - 10);
                     break;
                 }
                 else
                 {
                     if (fileList != "") fileList += "\n";
-                    fileList += Paths.ReplacePlatformPathText(filesToBeOverwritten[i]);
+                    fileList += Paths.HidePlatformInternalPath(filesToBeOverwritten[i]);
                 }
             }
             confirmDialog.Show(
-                Locale.GetStringAndFormat(
+                Locale.GetStringAndFormatIncludingPaths(
```

Comment:
We already called `HidePlatformInternalPath` on every file in the message, so there's no need to call it again.